gateway/src/node/storage/mod.rs

```diff
@@ -164,10 +204,10 @@ pub trait Storage: Send + Sync {
     ///
     /// # Arguments
     ///
-    /// * `blinded_serial_number`: the unique blinded serial number embedded in the credential
+    /// * `blinded_serial_number`: the unique blinded serial number embedded in the credential in base58 form
     async fn contains_credential(
         &self,
-        blinded_serial_number: &BlindedSerialNumber,
+        blinded_serial_number: String,
```

Comment:
can we not take it by reference?

Reply:
TO FIX

Reply:
Fixed in 6c5557c22fda73c4a027b13dd876de2e07bdfef6